code/script.js

```diff
@@ -231,95 +233,138 @@ const setSecret = () => {
 const start = () => {
   // Here we're setting charactersInPlay array to be all the characters to start with
   charactersInPlay = CHARACTERS
-  // What else should happen when we start the game?
+  generateBoard()
+  setSecret()
```

Comment:
Where is the function that sets the secret? Your chosen secret person is not the same as the one thats left when you've filtered everyone else out. You need the to both set the variable and the function so it works with the game. 
```
const setSecret = () => {
  secret = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)]
}
```